driver/camera.c

```diff
@@ -1176,7 +1176,7 @@ esp_err_t camera_init(const camera_config_t* config)
     }
 
     //ToDo: core affinity?
-    if (!xTaskCreatePinnedToCore(&dma_filter_task, "dma_filter", 4096, NULL, 10, &s_state->dma_filter_task, 1)) {
+    if (!xTaskCreatePinnedToCore(&dma_filter_task, "dma_filter", 4096, NULL, 10, &s_state->dma_filter_task, 0)) {
```

Comment:
see I left a todo note :) we should add this to menuconfig and have it as setting there. I think it's kinda important to be selectable.

Reply:
Yes, there should be an option to select the core or just let it go selecting the propre one, like httpd

Reply:
I am thinking of 3 options for maximum flexibility: No-Affinity, Core0, Core1